tests/modeltests/tst_models.cpp

```diff
@@ -161,6 +161,56 @@ const SymbolData prettifySymbolData[] = {
     {"std::unordered_multimap<int, float>",
      "std::unordered_multimap<int, float, std::hash<int>, std::equal_to<int>,"
      " std::allocator<std::pair<int const, float> > >"},
+    {"std::__function::__func<std::__bind<bool (foobar::map::api_v2::DeltaAccessImpl::*)"
```

Comment:
could / should __function::_func and __bind be prettified too? std::function and std::bind?

Reply:
also, please add more coverage for std::future, I think it's not prettified either (see my other comment and check the symbols in the cpp-parallel example)

Reply:
My plan was to try to get the basic support in place first and once that has landed work on adding support for more symbols. The ones starting with underscore are a bit special since they are internal symbols that differ between libstdc++ and libc++ so they might require a bit more thought. Is that ok?

Reply:
I added a prettification for std::allocator which I think fixes the std::function case.